Description: Apply youtube fix from upstream issue 7900
   * Fix for youtube failure #809454
Author: Wookey <wookey@debian.org>

Origin: upstream, https://github.com/rg3/youtube-dl/commit/b95779be21929c1aed1c0edbc3f9a5d11826973f
Bug: https://github.com/rg3/youtube-dl/issues/7900
Bug-Debian: https://bugs.debian.org/809454
Forwarded: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=809454#3
Last-Update: 2016-01-07

--- youtube-dl-2015.11.27.1.orig/youtube_dl/jsinterp.py
+++ youtube-dl-2015.11.27.1/youtube_dl/jsinterp.py
@@ -232,7 +232,7 @@ class JSInterpreter(object):
     def extract_function(self, funcname):
         func_m = re.search(
             r'''(?x)
-                (?:function\s+%s|[{;]%s\s*=\s*function|var\s+%s\s*=\s*function)\s*
+                (?:function\s+%s|[{;,]%s\s*=\s*function|var\s+%s\s*=\s*function)\s*
                 \((?P<args>[^)]*)\)\s*
                 \{(?P<code>[^}]+)\}''' % (
                 re.escape(funcname), re.escape(funcname), re.escape(funcname)),
